Cargo.toml

```diff
@@ -13,3 +13,19 @@ members = [
 exclude = [
     "fluent-cli",
 ]
+
+[workspace.dependencies]
+criterion = "0.3"
+fluent-langneg = "0.13"
+futures = "0.3"
+iai = "0.1"
+intl_pluralrules = "7.0.1"
+rustc-hash = "1"
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
Suggestion: I don't think `features` should be enabled/disabled at the workspace level, they should be done at the crate level.